bukkit/src/main/java/dev/aurelium/auraskills/bukkit/loot/parser/ItemLootParser.java

```diff
@@ -117,4 +118,27 @@ protected int[] parseAmount(ConfigurationNode config) {
         return new int[]{1, 1};
     }
 
+    protected double[] parseDurability(ConfigurationNode config) {
+        double minDamage = 0.0;
+        double maxDamage = 0.0;
+
+        if (config.hasChild("damage")) {
+            boolean wasMap = false;
+            for (ConfigurationNode damage : config.node("damage").childrenList()) {
```

Comment:
Since each item can only have one set of min/max damage, we don't need to use a list of maps to define this but just a regular map. So instead of looping through childrenList just check if `config.node("damage").isMap()` and use `config.node("damage", "min").getDouble()`. And so you also don't need a wasMap flag just put the single damage in the else clause of the isMap check.

The yaml for this would look like
```yaml
loot:
  - type: item
    material: diamond_axe
    weight: 1
    amount: 1
    damage:
      min: 0.5
      max: 0.95
```